src/sysinfo/nvapi_output.h

```diff
@@ -14,6 +14,9 @@ namespace dxvk {
 
         struct ColorData {
             bool HasST2084Support;
+
+            uint32_t BitsPerColor;
```

Comment:
I would prefer to declare `BitPerColor` here as `NV_BPC` and move the conversion from `NvAPI_Disp_HdrColorControl()` to `NvapiOutput::Initialize()`.